boards/arm/cy8ckit_062s4/doc/index.rst

```diff
@@ -0,0 +1,115 @@
+.. _cy8ckit_062s4:
+
+[INFINEON PSoC 62S4 Pioneer Kit]
+################################
+
+Overview
+********
+The PSOC 62S$ Pioneer kit has a CY8C62x4 MCU, which is an ultra-low-power PSoC device specifically designed for battery-operated analog
```

Comment:
62S$ --> 62S4
not a big deal but @DaWei8823, if you ever get a chance to submit a fix... :)

Reply:
@kartben thanks for the catch, I have this PR https://github.com/zephyrproject-rtos/zephyr/pull/61456